cartography/intel/aws/ec2/instances.py

```diff
@@ -121,7 +121,7 @@ def _load_ec2_instance_tx(
         instance_state: str,
         current_aws_account_id: str,
         region: str,
-        update_tag: str,
+        update_tag: int,
```

Comment:
The update tag really should be either string or int.
https://lyft.github.io/cartography/dev/writing-intel-modules.html#handling-cartography-s-update-tag